test/cmdlineargs.jl

```diff
@@ -61,15 +61,6 @@ let exename = `$(Base.julia_cmd()) --precompiled=yes --startup-file=no`
     @test !success(`$exename -L`)
     @test !success(`$exename --load`)
 
-    # --cpu-target
-    # NOTE: this test only holds true if image_file is a shared library.
```

Comment:
It won't make sense anymore. This was testing that a cpu target different from the one used to compile sysimg will throw an error but it will no longer be the case.